projects/sdk/src/lib/silo/Transfer.test.ts

```diff
@@ -0,0 +1,101 @@
+import { TokenValue } from "src/TokenValue";
+import { getTestUtils, ACCOUNTS } from "src/utils/TestUtils/provider";
+import { Transfer } from "./Transfer";
+
+const { sdk, account, utils } = getTestUtils();
+
+jest.setTimeout(30000);
+
+describe("Silo Transfer", function () {
+  const transfer = new Transfer(sdk);
+  const BEAN_TOKEN = sdk.tokens.BEAN;
+
+  const testDestination = ACCOUNTS[1][1];
+
+  describe("Transfer sourced from single crate", () => {
+    beforeAll(async () => {
+      await utils.resetFork();
+
+      // make a deposit
+      await BEAN_TOKEN.approveBeanstalk(TokenValue.MAX_UINT256);
+      await utils.setBEANBalance(account, BEAN_TOKEN.amount(2000));
+      const deposit = sdk.silo.buildDeposit(BEAN_TOKEN, account);
+      deposit.setInputToken(BEAN_TOKEN);
+      const tx1 = await deposit.execute(BEAN_TOKEN.amount(500), 0.1);
+      await tx1.wait();
```

Comment:
FYI, there's now a simple sdk.silo.deposit() method that does all this for you, w/o the need for a builder, since the builder is mostly a UI optimization.

Reply:
OK will take a look.

Reply:
https://github.com/BeanstalkFarms/Beanstalk/pull/307/commits/62cd6dccf29437cf7eda4f2d8a0be936d397087e